transaction/core/src/membership_proofs/mod.rs

```diff
@@ -12,16 +10,14 @@ use crate::{
     tx::{TxOut, TxOutMembershipElement, TxOutMembershipHash, TxOutMembershipProof},
 };
 use alloc::vec::Vec;
-use blake2::digest::Update;
 use core::convert::TryInto;
-
-use mc_crypto_hashes::Blake2b256;
+use mc_crypto_hashes::{Blake2b256, Digest};
 use subtle::{Choice, ConditionallySelectable, ConstantTimeEq};
 
 mod errors;
-mod range;
-
 pub use errors::Error as MembershipProofError;
+
+mod range;
```

Comment:
I'd like to keep the `mod`s and `pub use` grouped together (that other crate was wrong, this was correct :-)). It makes it much easier to switch to the preferred crate-based export:
```
pub use crate::{
    errors::Error as MembershipProofError,
    range::{Range, RangeError},
};
```
See also #1346

Reply:
The reason it didn't work is it was `pub use {` instead of `pub use crate::{`